psi4/src/psi4/adc/adc.cc

```diff
@@ -45,53 +45,48 @@ ADCWfn::ADCWfn(SharedWavefunction ref_wfn, Options& options) : Wavefunction(opti
     module_ = "adc";
 
     std::vector<std::string> irreps_ = molecule_->irrep_labels();
-    aoccpi_ = new int[nirrep_];
-    boccpi_ = new int[nirrep_];
-    avirpi_ = new int[nirrep_];
-    bvirpi_ = new int[nirrep_];
+    aoccpi_ = nalphapi_ - frzcpi_;
+    boccpi_ = nbetapi_ - frzcpi_;
+    avirpi_ = nmopi_ - nalphapi_ - frzvpi_;
+    bvirpi_ = nmopi_ - nbetapi_ - frzvpi_;
```

Comment:
Aesthetic change
```suggestion
    avirpi_ = nmopi_ - frzvpi_ - nalphapi_;
    bvirpi_ = nmopi_ - frzvpi_ - nbetapi_;
```

Reply:
Because there's something wrong with the test suite, I'll make the change only if I need to issue a new commit for some other reason. I agree aesthetically, but it's not worth a delay to the PR getting in. The buggy ADC code is not long for this world anyways. 🙂